### DISCUSSION TITLE
Allow regular buttons on horizontal stack or let the regular button have the sticky property

### DISCUSSION THREAD

**Discussion by Johnthenerdboss** [Clooos/Bubble-Card#779] [Feature requests]
I like the idea of the footer but I want to put regular bubble button as the footer so it's not limited to just pop up menus. Is there any way to pull this off?

I've tried regular CSS positioning using sticky/relative/fixed etc but just couldn't get it to work right, although it does move the box.

**Comment by Soukyuu** [comment]
I'm currently doing this using mushroom chips, which do what I want *except for one thing*: always be at the bottom of the screen. That's exactly what horizontal stack does and I'm surprised the functionality is so limited in comparison. Why not offer the full "button" functionality within horizontal stack?

**Comment by mattague** [comment]
This would be so handy
